docker.mk

```diff
@@ -4,9 +4,11 @@ DOCKER_NAMESPACE = getcapsule
 
 DPDK_IMG = dpdk
 DPDK_DEVBIND_IMG = dpdk-devbind
+DPDK_MACHINE = corei7
```

Comment:
do we need to specify here? they are defaulted in the dockerfile and we are not changing them in makefile.